tests_api/Unit/ConfigurationTest.php

```diff
@@ -0,0 +1,493 @@
+<?php
```

Comment:
One tiny nit: can we make this `tests/api` instead of `tests_api`? We have other test branches in the `tests` folder like `tests/opcache` and `tests/xdebug` so `tests/api` would follow the existing flow. :)

Reply:
sure thing